### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ajinabraham**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,8 +11,13 @@ A Bloom filter is a space-efficient probabilistic data structure that is used to
 * Lua 5.1, Lua 5.2, or LuaJIT
 * [CMake (2.8.7+)](http://cmake.org/cmake/resources/software.html)
 
-### CMake Build Instructions
+### Install Lua Libraries
+    
+    #For Lua 5.2
+    sudo apt-get install liblua5.2-dev libluabind-dev
```
Comment (trink) [inline README.md]: libluabind-dev should not be necessary

Comment (trink) [inline README.md]: Also, any of the flavors of Lua are a prerequisite. I don't really want to maintain a list of package install commands for every platform/distribution.

**Comment by ajinabraham** [comment]
I got the following error when I tried as per the readme

`make` is throwing errors
```
ajin@OpSec:~/Desktop/webarmor/lua_bloom_filter/release$ make
[ 50%] Building C object CMakeFiles/bloom_filter.dir/lua_bloom_filter.c.o
/home/ajin/Desktop/webarmor/lua_bloom_filter/lua_bloom_filter.c:223:30: error: array type has incomplete element type
 static const struct luaL_reg bloom_filterlib_f[] =
                              ^
/home/ajin/Desktop/webarmor/lua_bloom_filter/lua_bloom_filter.c:231:30: error: array type has incomplete element type
 static const struct luaL_reg bloom_filterlib_m[] =
                              ^
/home/ajin/Desktop/webarmor/lua_bloom_filter/lua_bloom_filter.c: In function ‘luaopen_bloom_filter’:
/home/ajin/Desktop/webarmor/lua_bloom_filter/lua_bloom_filter.c:252:3: error: implicit declaration of function ‘luaL_register’ [-Werror=implicit-function-declaration]
   luaL_register(lua, NULL, bloom_filterlib_m);
   ^
/home/ajin/Desktop/webarmor/lua_bloom_filter/lua_bloom_filter.c: At top level:
/home/ajin/Desktop/webarmor/lua_bloom_filter/lua_bloom_filter.c:223:30: error: ‘bloom_filterlib_f’ defined but not used [-Werror=unused-variable]
 static const struct luaL_reg bloom_filterlib_f[] =
                              ^
/home/ajin/Desktop/webarmor/lua_bloom_filter/lua_bloom_filter.c:231:30: error: ‘bloom_filterlib_m’ defined but not used [-Werror=unused-variable]
 static const struct luaL_reg bloom_filterlib_m[] =
                              ^
cc1: all warnings being treated as errors
make[2]: *** [CMakeFiles/bloom_filter.dir/lua_bloom_filter.c.o] Error 1
make[1]: *** [CMakeFiles/bloom_filter.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by trink** [comment]
I will look into it but these libs are really targeted it 5.1 (5.2 is not tested or used and therefore shouldn't have been/be included in the list)

**Pull request closed without merging**